src/nix/devcontainer-feature.json

```diff
@@ -21,6 +21,11 @@
             "default": "",
```

Comment:
As we are adding a new Feature option, can we bump the `minor` version of the Feature?

https://github.com/devcontainers/features/blob/da7abcca2b5986a3c2cb3ddf62e1067d2fa2a19f/src/nix/devcontainer-feature.json#L3

Reply:
bump to 1.2.0